libr/core/cmd_anal.c

```diff
@@ -2779,6 +2780,24 @@ static int cmd_anal_fcn(RCore *core, const char *input) {
 			r_core_cmdf (core, "\"afs %s\"", data);
 			free (sig);
 			free (data);
+		} else if (input[2] == 'r') { // "afsr"
+			if (input[3] == ' ') {
+				const char *ntype = (input[4])
```

Comment:
unnecessary parenthesys. but.. if input[3] have a space, input4 should be != ' ' already, because commands are chopped before reaching this point